opencensus/trace/propagation/jaeger_format.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import re
+
+from opencensus.trace.span_context import SpanContext
+from opencensus.trace.trace_options import TraceOptions
+
+_TRACE_CONTEXT_HEADER_NAME = 'UBER_TRACE_ID'
```

Comment:
I'd suggest to name it `_UBER_HEAER_NAME` to avoid confusions

Reply:
Updated